### PR TITLE
Add vobject recipe

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: recipes/vobject/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - ics_diff = vobject.ics_diff:main
+    - change_tz = vobject.change_tz:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - python-dateutil >=2.4.0
```
Comment (jakirkham) [inline recipes/vobject/meta.yaml]: Probably can drop `python-dateutil` from the build dependencies.

**Comment by jakirkham** [comment]
One minor thing. Otherwise LGTM.

**Comment by proinsias** [comment]
@jakirkham – I've updated the file to reflect this edit.

**Comment by jakirkham** [comment]
Thanks @proinsias.

File: recipes/vobject/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{% set name = "vobject" %}
+{% set version = "0.9.3" %}
+{% set md5 = "fe5996008d5e4858d63156b37b43d59c" %}
```
Comment (jakirkham) [inline recipes/vobject/meta.yaml]: We should probably start moving to `sha256` for checksums, but this can easily be done in the feedstock.

**Pull request merged**